seqr/views/apis/data_manager_api.py

```diff
@@ -331,6 +333,21 @@ def load_rna_seq_sample_data(request, sample_guid):
     return create_json_response({'success': True})
 
 
+def _notify_phenotype_prioritization_loaded(project, tool, num_samples, user):
+    url = f'{BASE_URL}project/{project.guid}/project_page'
+    project_link = f'<a href={url}>{project.name}</a>'
+    email = (
+        f'This is to notify you that {tool.title()} data for {num_samples} sample(s) '
+        f'has been loaded in seqr project {project_link} by {user.get_full_name()}'
```

Comment:
I do not think we want to tell external users who is doing the data loading. Its always an internal data manger that they don't know, so I think its just a confusing extra piece of information. I would take out `by {user.get_full_name()}` from the email body and leave it as is

Reply:
ah this makes sense, taking out {user.get_full_name()}